app/controllers/api/v1x2/tags_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1x2
+    class TagsController < Api::V1x1::TagsController
+      def acceptable_params
```

Comment:
Since we made it `private` in the original V1x0, it should probably be private here too.